DESIGN_GUIDELINES.md

```diff
@@ -1165,6 +1226,87 @@ use the SecretValue type [_awslint:secret-token_].
 * No need to put every class in a separate file. Try to think of a
   reader-friendly organization of your source files.
 
+### Levels of Abstraction
+
+There are three different levels of constructs in this library, beginning with
+low-level constructs, which we call _CFN Resources_ (or L1, short for "level 1")
+or Cfn (short for CloudFormation) resources. These constructs directly represent
+all resources available in AWS CloudFormation. CFN Resources are periodically
+generated from the AWS CloudFormation Resource Specification. They are named
+**Cfn**_Xyz_, where _Xyz_ is name of the resource. For example, CfnBucket
+represents the AWS::S3::Bucket AWS CloudFormation resource. When you use Cfn
+resources, you must explicitly configure all resource properties, which requires
+a complete understanding of the details of the underlying AWS CloudFormation
+resource model.
+
+The next level of constructs, L2, also represent AWS resources, but with a
```

Comment:
We like to say that L2s offer "intent based APIs"

Reply:
Yes, it says "intent-based API" on the very next line.